esphome/components/switch/switch.cpp

```diff
@@ -20,6 +20,11 @@ void Switch::toggle() {
   ESP_LOGD(TAG, "'%s' Toggling %s.", this->get_name().c_str(), this->state ? "OFF" : "ON");
   this->write_state(this->inverted_ == this->state);
 }
+void Switch::control(bool state) {
+  ESP_LOGD(TAG, "'%s' Controlling %s. Was %s", this->get_name().c_str(), state ? "ON" : "OFF",
+           this->state ? "ON" : "OFF");
```

Comment:
We have a macro for this
```suggestion
  ESP_LOGD(TAG, "'%s' Controlling %s. Was %s", this->get_name().c_str(), ONOFF(state),
           ONOFF(this->state));
```